### PR TITLE
feat(DEV-11277): Implement the Monite Iframe Application

### PR DESCRIPTION
**Pull request by vkuprin**

### CODE REVIEW

File: packages/sdk-iframe/src/iframe-app.tsx [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+}) => {
+  const [apiUrl, setApiUrl] = useState<string>('');
+  const [entityId, setEntityId] = useState<string>('');
+
+  useEffect(() => {
+    try {
+      const initialize = async () => {
+        const config = await getConfig();
+        setApiUrl(config.api_url);
+        const id = await getEntityId(config.api_url, fetchToken);
```
Comment (vkuprin) [inline packages/sdk-iframe/src/iframe-app.tsx:126]: Maybe makes sense to refactor to react-query, not sure... @radist2s
```ts
const useEntityId = (
  apiUrl: string,
  fetchToken: () => Promise<{
    access_token: string;
    token_type: string;
    expires_in: number;
  }>
) => {
  return useQuery(['entityId', apiUrl], () => getEntityId(apiUrl, fetchToken));
};
```

File: packages/sdk-drop-in/src/common-app-element.tsx [hunk 2]
```diff
@@ -221,22 +245,43 @@ export class DropInElement extends HTMLElement {
...
-        />
-      </React.StrictMode>
-    );
+    if (this.name === NameProps.iframeApp) {
+      const props = {
+        ...attributesProperties,
+        ...slotProperties,
+      } as Omit<ComponentProps<typeof IframeApp>, 'rootElements'>;
+
+      this.reactAppRoot.render(
```
Comment (vkuprin) [inline packages/sdk-drop-in/src/common-app-element.tsx:254]: document.create Element

File: packages/sdk-drop-in/index.html [hunk 3]
```diff
@@ -15,6 +15,7 @@
   </head>
   <body>
     <monite-app
+      name-prop="dropIn"
```
Comment (radist2s) [inline packages/sdk-drop-in/index.html:18]: `name-prop` probably is not needed anymore.

File: packages/sdk-drop-in/monite-iframe-app-consumer.html [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+      .monite-iframe-app {
+        width: 100%;
+        height: calc(100% - 10px);
+        border: none;
+      }
+    </style>
+    <title>Monite App Widget Label</title>
+    <script>
+      // not used
+      async function fetchAuthToken() {
```
Comment (radist2s) [inline packages/sdk-drop-in/monite-iframe-app-consumer.html:32]: Could you please remove not used with which marked with `// not used`?

File: packages/sdk-drop-in/monite-iframe-app.html [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    </style>
+    <title>Monite App Widget Label</title>
+  </head>
+  <body>
+    <div id="root"></div>
+
+    <script src="/src/monite-iframe-app-html.tsx" type="module"></script>
+
+    <script type="module">
+      // not used
```
Comment (radist2s) [inline packages/sdk-drop-in/monite-iframe-app.html:22]: Would be cool to remove garbage code from this file too `// not used`

**Review by radist2s — CHANGES_REQUESTED** [review]

**Pull request closed without merging**